### PR TITLE
Add pension reporting for altinn

### LINKED ISSUE #314: OTP and penson to altinn
We need to send OTP to altinn in a-melding
[a-meldingen_v2_0.pdf](https://github.com/dodoas/lodo/files/633485/a-meldingen_v2_0.pdf)
We should add a node after loennOgGodtgjoerelse so it will look like
```xml
        <arbeidsgiveravgift>
          <loennOgGodtgjoerelse>
            <beregningskodeForArbeidsgiveravgift>generelleNaeringer</beregningskodeForArbeidsgiveravgift>
            <sone>1</sone>
            <avgiftsgrunnlagBeloep>144990</avgiftsgrunnlagBeloep>
            <prosentsatsForAvgiftsberegning>14.1</prosentsatsForAvgiftsberegning>
          </loennOgGodtgjoerelse>

          <tilskuddOgPremieTilPensjon>
            <beregningskodeForArbeidsgiveravgift>generelleNaeringer</beregningskodeForArbeidsgiveravgift>
            <sone>1</sone>
            <avgiftsgrunnlagBeloep>1490</avgiftsgrunnlagBeloep>
            <prosentsatsForAvgiftsberegning>14.1</prosentsatsForAvgiftsberegning>
          </tilskuddOgPremieTilPensjon>
        </arbeidsgiveravgift>
```
beregningskodeForArbeidsgiveravgift, sone and prosentsatsForAvgiftsberegning should be the same as  it is for loennOgGodtgjoerelse. 

When we send a report we should have a input where we could enter an amount that will be OTP. This values should be saved in db and linked to this report. 

This need to be added when we send again and inactivate when canceled.
Make a new column on altinnsalary.list where we have OTP amount

### PR DESCRIPTION
**Pull request by mladjo2505**
Closes #314

### CODE REVIEW

File: modules/altinnsalary/view/altinn1list.php [hunk 1]
```diff
@@ -44,6 +44,11 @@
 </form>
 <br /> <br />
 <form name="altinnsalary_search" action="<? print $_lib['sess']->dispatch ?>t=altinnsalary.altinn1list" method="post">
+<span>OTP:</span><br/>
+<span>Sone: 1</span><br/>
```
Comment (mladjo2505) [inline modules/altinnsalary/view/altinn1list.php]: Select the same as it is set for the company, not hardcoded

File: modules/altinnsalary/view/salarylist.php [hunk 2]
```diff
@@ -252,6 +252,10 @@
 
 <br/>
   <input type="hidden" name="altinnReport1_periode" value='<?print $_periode; ?>'>
+  <span>OTP:</span><br/>
+  <span>Sone: 1</span><br/>
```
Comment (mladjo2505) [inline modules/altinnsalary/view/salarylist.php]: Select the same as it is set for the company, not hardcoded

File: modules/altinnsalary/view/altinn1list.php [hunk 3]
```diff
@@ -44,6 +44,11 @@
 </form>
 <br /> <br />
 <form name="altinnsalary_search" action="<? print $_lib['sess']->dispatch ?>t=altinnsalary.altinn1list" method="post">
+<span>OTP:</span><br/>
```
Comment (mladjo2505) [inline modules/altinnsalary/view/altinn1list.php:47]: translate on final review

File: modules/altinnsalary/view/list.php [hunk 4]
```diff
@@ -53,6 +53,7 @@ function highlight(elementid){
       <th class='menu'>Periode</th>
       <th class='menu'>Sent kl</th>
       <th class='menu'>Rapportert</th>
+      <th class='menu align-right'>OTP Bel&oslash;p</th>
```
Comment (mladjo2505) [inline modules/altinnsalary/view/list.php:56]: translate on final review

File: modules/altinnsalary/view/salarylist.php [hunk 5]
```diff
@@ -252,6 +252,10 @@
 
 <br/>
   <input type="hidden" name="altinnReport1_periode" value='<?print $_periode; ?>'>
+  <span>OTP:</span><br/>
```
Comment (mladjo2505) [inline modules/altinnsalary/view/salarylist.php:255]: translate on final review

**Comment by MrdjanStojimirovic** [comment]
First round review: @MrdjanStojimirovic
Final review with: @stoivo

**Review by MrdjanStojimirovic — APPROVED** [review]
Looks good.

File: modules/altinnsalary/view/altinn1list.php (hunk 3 above)
Comment (mladjo2505) [reply modules/altinnsalary/view/altinn1list.php:47]: not needed

File: modules/altinnsalary/view/list.php (hunk 4 above)
Comment (mladjo2505) [reply modules/altinnsalary/view/list.php:56]: not needed

File: modules/altinnsalary/view/salarylist.php (hunk 5 above)
Comment (mladjo2505) [reply modules/altinnsalary/view/salarylist.php:255]: not needed

**Review by stoivo — APPROVED** [review]

**Pull request merged**